### ISSUE TITLE
[Development Call] - Tutorial on Docker for researchers

### ISSUE THREAD

**Issue by gbocsardi** [tilburgsciencehub/website#1128]
The idea for this article would be to showcase how you can use Docker as a researcher by walking you through a toy research project. The aim is to show how you can work and develop inside the container, using both Python and R, and also interacting with your own filesystem. I think this is a preliminary to #309.

## Outline
- How to install Docker on Win/Mac 
- How to create a base image (with Python and R installed)
- How to mount directories to container
- Introduce the research project - IRIS petal prediction
- Running a linear regression on the IRIS data using RStudio and R inside the container
- Creating a nice visualization using Python and VSCode
- Saving the visualization to your personal drive (interacting with the filesystem)

**Comment by Fernando-Iscar** [comment]
Hi @gbocsardi,

I was thinking about this idea. I just double checked our Docker topics and there would be some overlapping content, for instance, [this topic](https://tilburgsciencehub.com/topics/automation/replicability/docker/docker/) already covers installation for all operating systems, or [this one](https://tilburgsciencehub.com/topics/automation/replicability/docker/dockerhub/) covers the basics for Docker Hub (what is it, import and export an image). 

Hence, some of the points in your proposed outline can be a short sentence pointing the readers to these resources to avoid overlapping content. What do you think?

**Comment by gbocsardi** [comment]
I completely agree, I think tying it to the existing content as much as possible should work great!
